figures/repo_statistic_plots.py

```diff
@@ -14,11 +23,9 @@
 
 datafile_url = os.path.join(sunpy_paper.data_dir, "loc_vs_time.csv")
 data = pd.read_csv(datafile_url, parse_dates=True, index_col=0, comment="#")
-
-x = data.index
+x = pd.to_datetime(data.index, utc=True)
 y1 = data["code"].values / 10000.0
```

Comment:
this is divided by 10000 - so label should be tens of thousands as @wafels mentioned in riot

Reply:
or equivalently could divide by 1000, and leave the label as it is

Reply:
This is more of a style comment.  I've become more of a fan of putting light dotted grid lines (vertical and horizontal) in line plots.  I think it helps guide the eye. Would the plots in Figure 1 benefit from grid lines?  In the left and center plots, vertical grid lines might help guide users to see when SunPy versions were released.

Reply:
added horizontal grid lines. not sure how to add vertical grid lines since we have two axes (year or release) so didn't do that.